paasta_tools/marathon_serviceinit.py

```diff
@@ -138,22 +138,53 @@ def get_verbose_status_of_marathon_app(app):
     output.append("    App created: %s (%s)" % (str(create_datetime), humanize.naturaltime(create_datetime)))
     output.append("    Tasks:")
 
+    def get_datestring_for_date(date):
```

Comment:
Actually if we're removing all the output except for 'Alive' for health checks we don't really need it.